includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -522,6 +523,95 @@ public static function send_customizable_receipt_email( $enable, $order, $class
 		return false;
 	}
 
+	/**
+	 * Send the customizable cancellation email in addition to WooCommerce Subscription's default.
+	 * We still want to allow WCS to send its cancellation email since this targets the store admin.
+	 *
+	 * @param bool            $enable        Whether to send the default receipt email.
```

Comment:
It seems that there is no default _cancellation_ email. This comment and the notice in the UI should be updated:

<img width="1072" alt="image" src="https://github.com/Automattic/newspack-plugin/assets/7383192/2c93880b-513d-4e1e-b27d-1e807f352541">

Reply:
Good catch! Fixed in https://github.com/Automattic/newspack-plugin/pull/3056/commits/e5b442295492885b5d95f3994a60c2e15a9ea22c